code/modules/food_and_drinks/food/snacks_soup.dm

```diff
@@ -225,3 +225,22 @@
 	bonus_reagents = list("nutriment" = 4, "vitamin" = 6)
 	tastes = list("beet" = 1)
 	foodtype = VEGETABLES
+
+/obj/item/reagent_containers/food/snacks/soup/electron
+	name = "electron soup"
+	desc = "A gastronomic curiosity of ethereal origin. It is famed for the minature weather system formed over a properly prepared soup."
+	icon_state = "electronsoup"
+	list_reagents = list("nutriment" = 3, "liquidelectricity" = 5)
+	tastes = list("mushroom" = 1, "electrons" = 4, "shockingly good")
+	filling_color = "#CC2B52"
+	foodtype = VEGETABLES | TOXIC
+
+/obj/item/reagent_containers/food/snacks/soup/bungocurry
+	name = "bungo curry"
+	desc = "A spicy vegetable curry made with the humble bungo fruit, Exotic!"
+	icon_state = "bungocurry"
+	bonus_reagents = list("vitamin" = 5, "bungojuice" = 6)
```

Comment:
You forgot to add bungo juice from the same PR that updates donuts.

Reply:
I dont plan on having a nothing chem. From what I can tell on TG it does nothing of note other then just being a juice that can be drinked for RP